components/frontend/src/widgets/TableRowWithDetails.css

```diff
@@ -0,0 +1,3 @@
+.ui.basic.button.expandcollapse:not(:hover, :focus) {
+    box-shadow: 0 0 0 0px transparent;
```

Comment:
Does this affect print to pdf in any way?

Reply:
Good questions, but no.